packages/dashboard-core-plugins/src/panels/NotebookPanel.tsx

```diff
@@ -578,6 +646,48 @@ class NotebookPanel extends Component<NotebookPanelProps, NotebookPanelState> {
     }
   }
 
+  handleMinimap() {
+    if (this.editor) {
+      const { defaultNotebookSettings } = this.props;
+      this.editor.updateOptions({
+        minimap: { enabled: defaultNotebookSettings.isMinimapEnabled },
+      });
+    }
+  }
+
+  handleMinimapWrapper() {
+    const { settings, defaultNotebookSettings, saveSettings } = this.props;
+    const newSettings: WorkspaceSettings = {
+      ...settings,
+      defaultNotebookSettings: {
+        isMinimapEnabled: !defaultNotebookSettings.isMinimapEnabled,
+      },
+    };
+    saveSettings(newSettings);
+  }
+
+  handleWordWrap() {
+    if (this.editor) {
+      const { settings } = this.state;
+      let { wordWrap } = settings;
+
+      if (wordWrap === 'on') {
+        wordWrap = 'off';
+      } else {
+        wordWrap = 'on';
+      }
+      this.editor.updateOptions({
+        wordWrap,
+      });
+      this.setState(prevState => ({
+        settings: {
+          ...prevState.settings,
+          wordWrap,
+        },
+      }));
+    }
+  }
```

Comment:
As mentioned in another comment, this should be broken into two functions:
`updateEditorWordWrap`: Should take the word wrap from settings, and call `editor.updateOptions`. Call it from `componentDidUpdate`
`handleWordWrapChange`: Should just toggle the option in the state settings, doesn't need to set on the actual editor.